internal/events/token_pool_created.go

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2021 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package events
+
+import (
+	"github.com/hyperledger-labs/firefly/internal/log"
+	"github.com/hyperledger-labs/firefly/pkg/fftypes"
+	"github.com/hyperledger-labs/firefly/pkg/tokens"
+)
+
+func (em *eventManager) TokenPoolCreated(tk tokens.Plugin, pool *fftypes.TokenPool, signingIdentity string, additionalInfo fftypes.JSONObject) error {
+
+	log.L(em.ctx).Infof("-> TokenPoolCreated id=%s author=%s", pool.ID, signingIdentity)
```

Comment:
Cool to see this pattern for a method entry/exit trace going in.
However, I do think it should probably be `Debugf` rather than `Infof` - for info level, we should just have the one notification when the actual successful thing has happened

Reply:
This was definitely copied from somewhere; I'll revisit.

Reply:
Looks like I copied from `BatchPinComplete`